src/routes/index.svelte

```diff
@@ -68,34 +74,43 @@
   {#if brands}
     {#each brands as item}
 			{#if item.post}
-				<Card>
-					<PrimaryAction on:click={() => doAction('openItemPage')}>
-
+				<div style="position: relative;">
+					<Card>
 						<Media
 							style="background-image: url({getImageURL(item)});"
 							aspectRatio="16x9" />
-						<Content>
-							{#if item.phones}
-								<div class="phone-grid">
-									{#each item.phones as phone}
-										<Phone phoneNumber={phone} />
-									{/each}
-								</div>
-							{/if}
-							<p class="caption" on:click={() => item.isCaptionOpen = !item.isCaptionOpen}>
-								{#if item.isCaptionOpen}
-									{item.post.caption}
+							<div class="location-container">
+								{#if item.location}
```

Comment:
en pro de "Separation of concerns", en lugar de tener dos ramas para definir como llamar a "<Location", mejor solo tener una llamada pasándole la información necesaria para que el componente sea autosuficiente.

Reply:
no entiendo bien de que hablas

Reply:
Ahorita se tiene:

```
{#if item.location}
<Location ...
{:else}
<Location ...
{/if}
```
Esas "ramas" (branches: if-else) es mejor pasarlas a `<Location>`, que el componente tenga la responsabilidad de eso en lugar de `index`

Reply:
nevermind, ya entendí, simón, I'll do that